models/Offering.js

```diff
@@ -48,6 +48,7 @@ const OfferingSchema = Schema({
   status: {
     type: String,
     required: true,
+    default: "PENDINGS",
```

Comment:
PENDINGS with an "s" ? or its supposed to be "pending"

Reply:
Fixed